### DISCUSSION TITLE
Remote Control Plugin responses "Incompatible shuffle request version"

### DISCUSSION THREAD

**Discussion by offermannu** [Stellarium/stellarium#2017] [Q&A]
Hi there, 

I activated the Remote Control Plugin as described in the documentation.
Port is default at 8090, all on localhost.
After restarting Stellarium I can see the remote control icon in the lower toolbar - it's enabled.
See also the outputs in the log:
`
Loaded plugin "RemoteControl"
RemoteControl using QtWebApp version 1.6.4-stellarium
StaticFileController: docroot=/Applications/Stellarium.app/Contents/Resources/webroot, encoding=UTF-8, maxAge=60
StaticFileController: cache timeout=60000, size=1048576
`

Now when I open [http://localhost:8090/](http://localhost:8090/) in Firefox, Chrome or Safari I get the message

`Incompatible shuffle request version`.

that's it. I'm getting the same reesult when I try it with curl and some other URLs.

I'm using Stellarium 0.21.1
on macos 11.6 (20G165)

Best regards
Udo

**Comment by offermannu** [comment]
I changed the port and now it's working. Seems that the port 8090 is occupied by another server.

**Reply by gzotti** [reply]
Yes, just google for this error. It's not in our code.

**Comment by offermannu** [comment]
Absolutely 😉
But the Plugin could log an Error in thus case, couldn't it?

- Udo

Georg Zotti ***@***.***> schrieb am Mo., 8. Nov. 2021, 23:28:

**Comment by james7643872** [comment]
The port was changed, and it is now operational. It appears that another server is using port 8090. I checked from this [site](https://bagshopes.com/backpack-with-built-in-lunch-box/).
